src/main/java/com/gtnewhorizons/modularui/common/widget/textfield/BaseTextFieldWidget.java

```diff
@@ -41,6 +41,7 @@ public class BaseTextFieldWidget extends Widget implements IWidgetParent, Intera
 
     static {
         format.setGroupingUsed(false);
+        format.setMaximumFractionDigits(8);
```

Comment:
Wouldn't it make sense to allow configuring digits per widget? I mean adding setter method

Reply:
Is it possible to rework the formatter to be non static for this?

Reply:
Oh I forgot about that. Deprecate static one and add non-static variable `decimalFormat`

Reply:
Some methods reference `format` from a static context so I left them untouched. However I tested in TT to see if I can set the maximum fraction digits and that worked.